e2e/client/playwright/authoring.spec.ts

```diff
@@ -38,3 +38,28 @@ test('can apply "populate abstract" macro', async ({page}) => {
         page.locator(s('authoring', 'authoring-field=abstract')).getByRole('textbox'),
     ).toHaveText('test sport story body');
 });
+
+test('can embed if url is valid', async ({page}) => {
+    await restoreDatabaseSnapshot();
+
+    const monitoring = new Monitoring(page);
+
+    await page.goto('/#/workspace/monitoring');
+
+    await monitoring.selectDesk('Sports');
+
+    await page.locator(
+        s('monitoring-group=Sports / Working Stage', 'article-item=test sports story'),
+    ).dblclick();
+
+    await expect(
+        page.locator(s('authoring', 'authoring-field=body_html')).getByRole('textbox'),
+    ).toHaveText('test sport story body');
+
+    await page.locator(s('Embed')).getByRole('button').click();
+
+    await page.getByPlaceholder('Enter URL or code to embed').type('https://google.com');
+
+    await page.locator(s('embed-controls', 'submit')).click();
+    await expect(page.getByText('https://google.com')).toBeDefined();
```

Comment:
This doesn't seem to be testing much except that you enter the URL and later have the URL present in the page. I'd think embedding does more than that?